package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "service",
-  "version": "0.1.1",
+  "version": "1.1.0",
```

Comment:
Need to update package-lock.json to be in sync. Can use [npm version](https://docs.npmjs.com/cli/v8/commands/npm-version) to do it.

Reply:
Thanks.  That worked great.